src/index.css

```diff
@@ -23,23 +8,23 @@ body {
 }
 
 h1 {
-  font-size: 3.2em;
+  font-size: var(--font-size-mammoth);
   line-height: 1.1;
 }
 
 button {
```

Comment:
Remove the styles for buttons. We are going to use Components for buttons instead of making global utility classes.